src/common/logging/Logging.c

```diff
@@ -133,9 +133,9 @@ void SetMaxLogSizeDebugMultiplier(unsigned int value)
     g_maxLogSizeDebugMultiplier = value;
 }
 
-static int RestrictFileAccessToCurrentAccountOnly(const char* fileName)
+static int RestrictLogFileAccess(const char* fileName)
```

Comment:
> RestrictLogFileAcces [](http://example.com/codeflow?start=10&length=21)

Please do not do this change -- it affects all of OSConfig stack, not just the AzPolicy related.

Please close this PR and reach to OSConfig team to discuss this, why is needed, and together find a solution